doc/reference/bluetooth/mesh/core.rst

```diff
@@ -21,6 +21,39 @@ the LPN API allows the application to trigger the polling at any time through
 interval, poll event timing and Friend requirements is controlled through the
 :option:`CONFIG_BT_MESH_LOW_POWER` option and related configuration options.
 
+Replay Protection List
+**********************
+
+Each mesh message is encrypted and authenticated using application key and
```

Comment:
Some of the statements here are not fully accurate as there is more nuance attached to most of them. I suggest removing this paragraph.